editor/editor_inspector.cpp

```diff
@@ -2772,10 +2778,8 @@ void EditorInspector::update_tree() {
 				array_element_prefix = p.class_name;
 				editor_inspector_array = memnew(EditorInspectorArray);
 
-				String array_label = path.contains("/") ? path.substr(path.rfind("/") + 1) : path;
-				array_label = EditorPropertyNameProcessor::get_singleton()->process_name(property_label_string);
 				int page = per_array_page.has(array_element_prefix) ? per_array_page[array_element_prefix] : 0;
-				editor_inspector_array->setup_with_move_element_function(object, array_label, array_element_prefix, page, c, use_folding);
+				editor_inspector_array->setup_with_move_element_function(object, property_label_string, array_element_prefix, page, c, use_folding);
```

Comment:
I removed `array_label` here because it was immediately overwritten on the next line, and `property_label_string` is already processed earlier so processing it again does nothing.

Reply:
I suspect if this is a typo, and should be processing `array_label` instead of `property_label_string`.

Reply:
I tried replacing `property_label_string` with `array_label` on line 2776, but somehow that breaks it and makes the array use the group name if it's inside a group.

![image](https://user-images.githubusercontent.com/67974470/160221941-63297213-f045-4278-9ea2-44a2cca5ad49.png)

I removed everything dealing with arrays from this PR, now it only handles groups. I guess that can be done in another PR if needed.